LocalPreferences.toml

```diff
@@ -0,0 +1,9 @@
+[MPIPreferences]
```

Comment:
Did you want to submit this file?

Reply:
No, I didn't. I removed it again. Thanks!